omnigibson/objects/dataset_object.py

```diff
@@ -209,6 +209,29 @@ def _initialize(self):
         # Run super method first
         super()._initialize()
 
+        # Apply any forced light intensity updates.
+        if gm.FORCE_LIGHT_INTENSITY is not None:
+            def recursive_light_update(child_prim):
+                if "Light" in child_prim.GetPrimTypeInfo().GetTypeName():
+                    child_prim.GetAttribute("intensity").Set(gm.FORCE_LIGHT_INTENSITY)
+
+                for child_child_prim in child_prim.GetChildren():
+                    recursive_light_update(child_child_prim)
+
+            recursive_light_update(self._prim)
+
+        # Apply any forced roughness updates
+        if gm.FORCE_ROUGHNESS is not None:
+            def recursive_roughness_update(child_prim):
+                if child_prim.GetPrimTypeInfo().GetTypeName() == "Shader":
```

Comment:
oh this should be done more programmatically, like so:

```python
for mat in self.materials:
    mat.reflection_roughness_constant = gm.FORCE_ROUGHNESS
    mat.reflection_roughness_texture_influence = 0.0
```

Ideally it'd be the same for lights but i haven't had time to that yet.